includes/widgets/image-carousel.php

```diff
@@ -88,9 +88,13 @@ protected function is_dynamic_content(): bool {
 	 * @return array Widget style dependencies.
 	 */
 	public function get_style_depends(): array {
-		return $this->load_widgets_styles_in_head()
-			? [ 'widget-image-carousel' ]
-			: [];
+		$assets = [ 'e-swiper', 'swiper' ];
+
+		if ( $this->load_widgets_styles_in_head() ) {
+			$assets[] = 'widget-image-carousel';
+		}
+
+		return $assets;
```

Comment:
It's a minor change, but this is the convention:

```suggestion
		$style_depends = [ 'e-swiper', 'swiper' ];

		if ( $this->load_widgets_styles_in_head() ) {
			$style_depends[] = 'widget-image-carousel';
		}

		return $style_depends;
```